services/movie.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
```

Comment:
+ Optional typing

Reply:
Do you mean make it like that `genres_ids: Optional[list[int]]` instead of  `genres_ids: list[int] = None`?